### DISCUSSION TITLE
Error: Command failed with exit code 1: npx nuxt build --standalone --no-lock --config-file "nuxt.config.js" /vercel/path0

### DISCUSSION THREAD

**Discussion by nicolaselielll** [vercel/community#1572] [Help]
### Summary

I'm getting this error on my nuxt project: Error: Command failed with exit code 1: npx nuxt build --standalone --no-lock --config-file "nuxt.config.js" /vercel/path0

I have tried:

- Changing node.js version
- Reinstalling & removing modules
- Using different versions of `@nuxtjs/vercel-builder`
- Defining different versions of vercel-builder module to vercel.json
- Deploying on a different Vercel project
- Deploying from a new repo

The error occured when I cloned the deployed code to my local machine & after only optimizing images and not touching the code at all.

Is there anything I could do to fix this?

Here's my package.json:

```
{
  "name": "project-name",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate"
  },
  "engines": {
    "node": "16.x"
  },
  "dependencies": {
    "@nuxtjs/axios": "^5.12.2",
    "@nuxtjs/vuetify": "^1.11.2",
    "core-js": "^3.6.5",
    "dotenv": "^16.0.3",
    "gsap": "npm:@gsap/shockingly@^3.10.4",
    "nuxt": "^2.14.6",
    "nuxt-gsap-module": "^1.7.1",
    "v-video-embed": "^1.0.8"
  },
  "devDependencies": {
    "@nuxtjs/vercel-builder": "^0.22.1"
  }
}
```

This is my vercel.json:

```
{
  "version": 2,
  "builds": [
      {
        "src": "nuxt.config.js",
        "use": "@nuxtjs/vercel-builder",
        "config": {}
      }
    ]
  }
```

**Comment by iampapagray** [comment]
Did you find any solution?
